lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -1267,6 +1267,23 @@ SPIRVToLLVMDbgTran::transTypeTemplateParameter(const SPIRVExtInst *DebugInst) {
                                                              false);
 }
 
+// Workaround for the incorrect enum used for OpTypeTemplateTemplateParameter
+// and OpTypeTemplateParameterPack instructions previously. The W/A added
+// to enforce compatibility with previously generated SPIR-V modules.
+DINode *SPIRVToLLVMDbgTran::transTypeTemplateParameterInst(
```

Comment:
This code is only tested locally. Ideally we should add an infrastructure to test pre-compiled binaries to ensure backward compatibility (that was probably also discussed during tooling wg call). I'll open an issue in github for this.